data/json/recipes/other/parts_construction.json

```diff
@@ -18,17 +18,15 @@
     "type": "recipe",
     "activity_level": "fake",
     "result": "wood_beam",
-    "id_suffix": "from logs",
```

Comment:
These should be new recipes, not replacing the old ones. People still need the option to use hand tools.

Reply:
Ok, I reverted changes and added my recipes as alternatives